### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by 4xmplme**
- [DEMO LINK](https://4xmplme.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://4xmplme.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style/card.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.card {
+  padding: 16px;
```
Comment (Viktor-Kost) [inline src/style/card.css:2]: apply bigger padding-top to a card and remove margin-top from an image

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,60 @@
...
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
```
Comment (Viktor-Kost) [inline src/index.html:39]: no needs to add an empty line between _oneline_ siblings. Only between _multiline_ ones
```suggestion
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
```

File: src/style/card.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.card__title {
+  margin: 40px 0 0;
```
Comment (Viktor-Kost) [inline src/style/card.css:16]: just margin-top. Update for similar cases 
```suggestion
  margin-top: 40px;
```

File: src/style/card.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+.card__product-code {
+  margin: 4px 0 0;
+  color: var(--secondary-color);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__review {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (Viktor-Kost) [inline src/style/card.css:32]: redundant. Remove at similar cases
```suggestion
```

File: src/style/stars.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (Viktor-Kost) [inline src/style/stars.css:10]: use background `shorthand`

File: src/style/index.css [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-style: normal;
```
Comment (Viktor-Kost) [inline src/style/index.css:15]: it's a default value
```suggestion
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Good job!
Some minor fixes needed

**Review by vlad-demchuk — APPROVED** [review]

**Pull request closed without merging**